### ISSUE TITLE
Unable to install blosc with pip version 20.X.X (Python 3.8.1)

### ISSUE THREAD

**Issue by MomIsBestFriend** [scikit-build/scikit-build#467]
(Feel free to __close this issue if you're not supporting python 3.8.1__, and sorry for your time)

I tried both cases on a fresh virtual enviroments, the only thing that was changed was the pip version (versions 20.0 and 20.0.1 gave the same error)

Trying to install ```blosc``` with ```pip 19.3.1``` runs without an error.

But when I tried to install ```blosc``` with ```pip 20.0.1``` I got this error message:
```
$ python -m pip install blosc
Collecting blosc
  Using cached blosc-1.8.3.tar.gz (809 kB)
  Installing build dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: /home/User/Venvs/venv/bin/python /home/User/Venvs/venv/lib/python3.8/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-06mk_dm7/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel scikit-build cmake ninja
       cwd: None
  Complete output (19 lines):
  Collecting setuptools
    Using cached setuptools-45.1.0-py3-none-any.whl (583 kB)
  Collecting wheel
    Using cached wheel-0.33.6-py2.py3-none-any.whl (21 kB)
  Collecting scikit-build
    Using cached scikit_build-0.10.0-py2.py3-none-any.whl (66 kB)
  Collecting cmake
    Using cached cmake-3.13.2.post1.tar.gz (27 kB)
      ERROR: Command errored out with exit status 1:
       command: /home/User/Documents/Github/Community/Python/Venvs/venv/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-h7mbzckq/cmake/setup.py'"'"'; __file__='"'"'/tmp/pip-install-h7mbzckq/cmake/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-h7mbzckq/cmake/pip-egg-info
           cwd: /tmp/pip-install-h7mbzckq/cmake/
      Complete output (5 lines):
      Traceback (most recent call last):
        File "<string>", line 1, in <module>
        File "/tmp/pip-install-h7mbzckq/cmake/setup.py", line 7, in <module>
          from skbuild import setup
      ModuleNotFoundError: No module named 'skbuild'
      ----------------------------------------
  ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  ----------------------------------------
ERROR: Command errored out with exit status 1: /home/User/Venvs/venv/bin/python /home/User/Venvs/venv/lib/python3.8/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-06mk_dm7/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel scikit-build cmake ninja Check the logs for full command output.
```

Additional information that might help:

<details>

INSTALLED VERSIONS
------------------
python : 3.8.1.final.0
python-bits  : 64
OS               : Linux
OS-release       : 5.4.13.a-1-hardened
machine          : x86_64
byteorder        : little
LC_ALL           : None
LANG             : en_US.UTF-8
LOCALE           : en_US.UTF-8

setuptools : 45.1.0
wheel : None

</details>

**Comment by thewtex** [comment]
Hi,

Does the build complete if

```
$ python -m pip install scikit-build
```

is executed *before*:

```
$ python -m pip install blosc
```

?

**Comment by MomIsBestFriend** [comment]
Hi @thewtex , sorry for the delayed response, this issue is resolved with ```pip``` version ```20.0.2```.

Reopen this issue if you like to investigate further, why the build was failing with the ```pip``` versions of ```20.0.0``` and ```20.0.1```, I'd be sure happy to help :)

Thank you for your time :)
